homeassistant/components/emulated_hue/__init__.py

```diff
@@ -244,6 +249,10 @@ def __init__(self, hass, conf):
             if hidden_value is not None:
                 self._entities_with_hidden_attr_in_config[entity_id] = hidden_value
 
+        # Get whether all non-dimmable lights should be reported as dimmable
+        # for compatibility with older installations.
+        self.lights_all_dimmable = conf.get(CONF_LIGHTS_ALL_DIMMABLE)
```

Comment:
Looks like we are stuck with `conf.get` because the *preexisting* tests aren't written in the standard way.